### PR TITLE
add Ad-astra bridge docs

### PR DESCRIPTION
**Pull request by dragos-rebegea**
#### Did you test the changes locally ?
- [ ] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/bridge/transfer-flows.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+id: transfer-flows
+title: Transfer Flows
+---
+
+### Ethereum to MultiversX
+1. A user deposits the ERC20 tokens that they want to transfer to the MultiversX network on the **Safe(1)** contract.
+2. The **Safe(1)** contract groups multiple deposits into batches.
+3. After a certain period of time, each batch becomes final and is processed by the relayers.
+4. The relayers propose, vote, and perform the transfer using the **Bridge (4)** contract with a consensus of 7/10 votes.
+5. The user receives the equivalent amount of ESDT tokens on their recipient address on the MultiversX network.
```
Comment (ovidiuolteanu) [inline docs/bridge/transfer-flows.md:11]:
```suggestion

```

File: docs/bridge/transfer-flows.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+6. On the MultiversX network, the same amount of ESDT tokens are minted as were deposited on the Ethereum network.
```
Comment (ovidiuolteanu) [inline docs/bridge/transfer-flows.md:12]:
```suggestion
5. On the MultiversX network, the same amount of ESDT tokens are minted as were deposited on the Ethereum network.
6. The user receives the equivalent amount of ESDT tokens on their recipient address on the MultiversX network.
```

Comment (ovidiuolteanu) [reply docs/bridge/transfer-flows.md:12]: Swapped these lines as it would more sense due to the logical flow.

**Review by ovidiuolteanu — CHANGES_REQUESTED** [review]

**Review by ovidiuolteanu — DISMISSED** [review]

File: docs/bridge/architecture.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Ad-Astra Bridge is a system that allows for the transfer of ERC20 tokens between the Ethereum and MultiversX networks. The system is composed of several contracts and relayers that work together to facilitate the transfer of tokens.
+
+## Ethereum Contracts
+- **Repo**: https://github.com/multiversx/mx-bridge-eth-sc-sol
+- **Safe (1)**: A contract that allows users to deposit ERC20 tokens that they want to transfer to the MultiversX network.
+- **Bridge(2)**: A contract that facilitates the transfer of tokens from Ethereum to MultiversX.
+
+## MultiversX Contracts
+- **Repo**: https://github.com/multiversx/mx-bridge-eth-sc-rs
+- **Safe (3)**: A contract that allows users to deposit ESDT tokens that they want to transfer back to the Ethereum network.
```
Comment (bogdan-rosianu) [inline docs/bridge/architecture.md:15]: remove 'back'

File: docs/bridge/transfer-flows.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+id: transfer-flows
+title: Transfer Flows
+---
+
+### Ethereum to MultiversX
```
Comment (bogdan-rosianu) [inline docs/bridge/transfer-flows.md:6]: `##` instead of `###`

File: docs/bridge/transfer-flows.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+1. A user deposits the ERC20 tokens that they want to transfer to the MultiversX network on the **Safe(1)** contract.
+2. The **Safe(1)** contract groups multiple deposits into batches.
+3. After a certain period of time, each batch becomes final and is processed by the relayers.
+4. The relayers propose, vote, and perform the transfer using the **Bridge (4)** contract with a consensus of 7/10 votes.
+5. On the MultiversX network, the same amount of ESDT tokens are minted as were deposited on the Ethereum network.
+6. The user receives the equivalent amount of ESDT tokens on their recipient address on the MultiversX network.
+
+### MultiversX to Ethereum
```
Comment (bogdan-rosianu) [inline docs/bridge/transfer-flows.md:14]: `##` instead of `###`

File: docs/bridge/transfer-flows.md (hunk 4 above)
Comment (dragos-rebegea) [reply docs/bridge/transfer-flows.md:6]: done

File: docs/bridge/transfer-flows.md (hunk 5 above)
Comment (dragos-rebegea) [reply docs/bridge/transfer-flows.md:14]: done

File: docs/bridge/architecture.md (hunk 3 above)
Comment (dragos-rebegea) [reply docs/bridge/architecture.md]: removed

**Review by bogdan-rosianu — APPROVED** [review]

**Review by ovidiuolteanu — APPROVED** [review]

**Pull request merged**